crates/egui/src/style.rs

```diff
@@ -212,6 +212,9 @@ pub struct Style {
     ///
     /// This only affects a few egui widgets.
     pub explanation_tooltips: bool,
+
+    /// If true and scrolling is enabled for only one direction, allow horizontal scrolling without pressing shift
+    pub always_scroll_the_only_direction: bool,
```

Comment:
I'm not sure we we should have this on by default as it will break existing apps in subtle way: a hscroll inside a vscroll (or the other way around) will no longer work as expected (the inner scroll area will steal all input)